dashboard/v1.1/src/pages/Config/Config.tsx

```diff
@@ -84,10 +84,8 @@ const Config: FC<{}> = () => {
           }
         });
       });
-      let routename =
-        route[0].substring(0, 40) + (route[0].length > 40 ? '...' : '');
       tableData.push({
-        route: routename,
+        route: route[0],
```

Comment:
Because that is what an Array.from() returns from a Map(). We could basically do an extra step iterate through all the values of the map and use another variable to store them in a better way, if you suggest